pkg/cloud/manifestwork/machineset.go

```diff
@@ -82,7 +82,7 @@ func (msd *manifestWorkMachineSetDeployer) Deploy(machineSet *unstructured.Unstr
 	}, msd.eventRecorder)
 }
 
-func (msd *manifestWorkMachineSetDeployer) GetWorkerNodeImage(machineSet *unstructured.Unstructured, infraID string) (string, error) {
+func (msd *manifestWorkMachineSetDeployer) GetWorkerNodeImage(workerNodeList []string, machineSet *unstructured.Unstructured, infraID string) (string, error) {
```

Comment:
The function parameters changed in cloud-prepare https://github.com/submariner-io/cloud-prepare/blob/devel/pkg/ocp/machinesets.go#L72 by